ai_dungeon_cli/__init__.py

```diff
@@ -114,10 +114,12 @@ def init_configuration_file(self):
 
         if exists(cfg, "prompt"):
             self.prompt = cfg["prompt"]
-
-        self.auth_token = cfg["auth_token"]
-        self.email = cfg["email"]
-        self.password = cfg["password"]
+        if exists(cfg, "auth_token"):
+            self.auth_token = cfg["auth_token"]
+        if exists(cfg, "email"):
+            self.email = cfg["email"]
+        if exists(cfg, "password"):
+            self.password = cfg["password"]
```

Comment:
I've never seen this notation.

I've always encountered:

```
if "password" in cfg:
    self.password = cfg["password"]
```

Which seems like the standard way to test for an element in a dict.

Reply:
My bad, didn't realize `exists()` was defined above.

Not to great to do a code review on a cell phone, eh!